projects/ngx-sub-form/src/lib/ngx-automatic-root-form.component.ts

```diff
@@ -0,0 +1,40 @@
+import { NgxRootFormComponent } from './ngx-root-form.component';
+import { Observable, timer } from 'rxjs';
+import { debounce } from 'rxjs/operators';
+import { isEqual } from 'lodash-es';
+
+export const NGX_SUB_FORM_HANDLE_DATA_OUTPUT_STRATEGIES = {
+  debounce: (time: number): ReturnType<NgxAutomaticRootFormComponent<any, any>['handleDataOutput']> => obs =>
+    obs.pipe(debounce(() => timer(time))),
+};
+
+export abstract class NgxAutomaticRootFormComponent<
+  ControlInterface,
+  FormInterface = ControlInterface
+> extends NgxRootFormComponent<ControlInterface, FormInterface> {
+  public ngOnInit(): void {
+    super.ngOnInit();
```

Comment:
This is dangerous and should be handled differently as this will call `registerOnChange` an we only want to call it few lines later

Reply:
Fixed.